sentry/README.md

```diff
@@ -50,35 +58,57 @@ one should use the [`sentry-core`] crate instead.
-
-Additional features:
-
-* `anyhow`: Enables support for the `anyhow` crate.
-* `debug-images`: Attaches a list of loaded libraries to events (currently only supported on unix).
-* `error-chain`: Enables support for the `error-chain` crate.
-* `failure`: Enables support for the `failure` crate.
-* `log`: Enables support for the `log` crate.
-* `env_logger`: Enables support for the `log` crate with additional `env_logger` support.
-* `slog`: Enables support for the `slog` crate.
-* `test`: Enables testing support.
-* `debug-logs`: Uses the `log` crate for internal logging.
-* `reqwest`: Enables the `reqwest` transport, which is currently the default.
-* `curl`: Enables the curl transport.
-* `surf`: Enables the surf transport.
-* `native-tls`: Uses the `native-tls` crate, which is currently the default.
-  This only has an effect on the `reqwest` transport.
-* `rustls`: Enables the `rustls` support of the `reqwest` transport.
-  Please note that `native-tls` is a default feature, and one needs to use
-  `default-features = false` to completely disable building `native-tls` dependencies.
+Additional functionality and integrations are enabled via feature flags. Some features require
+extra setup to function properly. 
+
+| Feature | Default | Is Integration | Deprecated | Additional notes |
+| --- | --- | --- | --- | --- |
+| `backtrace` | ✅ | 🔌 | | |
+| `contexts` | ✅ | 🔌 | | |
+| `panic` | ✅ | 🔌 | | |
+| `transport` | ✅ | | | |
+| `anyhow` | | 🔌 | | |
+| `test` | | | | |
+| `debug-images` | | 🔌 | | |
+| `log` | | 🔌 |  |Requires additional setup; See [`sentry-log`]'s documentation. |
+| `debug-logs` | | | ✂️ | | Requires additional setup; See [`sentry-log`]'s documentation. |
+| `slog` | | 🔌 | | | Requires additional setup; See [`sentry-slog`]'s documentation. |
+| `reqwest` | ✅ | | | |
+| `native-tls` | ✅ | | | `reqwest` must be enabled. | |
+| `rustls` | | | | `reqwest` must be enabled. `native-tls` must be disabled via `default-features = false`. | |
+| `curl` | | | | |
+| `surf` | | | | |
+
+[`sentry-log`]: https://docs.rs/sentry-log
+[`sentry-slog`]: https://docs.rs/sentry-slog
+
+Descriptions of all features, grouped by an arbitrary set of categories, sorted in a similarly 
+arbitrary order:
+
+### Default features
+- `backtrace`: Enables backtrace support.
+- `contexts`: Enables capturing device, OS, and Rust contexts.
+- `panic`: Enables support for capturing panics.
+- `transport`: Enables the default transport, which is currently `reqwest` with `native-tls`.
+
+### Debugging/Testing
+- `anyhow`: Enables support for the `anyhow` crate.
+- `test`: Enables testing support.
+- `debug-images`: Attaches a list of loaded libraries to events (currently only supported on Unix).
+
+### Logging
+- `log`: Enables support for the `log` crate.
+- `slog`: Enables support for the `slog` crate.
+- `debug-logs`: **Deprecated**. Uses the `log` crate for internal logging.
```

Comment:
not really deprecated

Reply:
that would be nice.  but we should run it in CI in that case

Reply:
just so i have this sorted out: sentry-core's `debug-logs` feature is deprecated according to its docs: https://github.com/getsentry/sentry-rust/blob/master/sentry-core/README.md#features

`debug-logs` on sentry specifies sentry-core's `debug-logs` as a dependency. is `debug-logs` on sentry-core deprecated, but not on sentry itself?